CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 * In general follow (https://docs.npmjs.com/getting-started/semantic-versioning) versioning.
 
 ## next
+
+## 1.2.2
```

Comment:
When adding features, minor version should be increased.

Reply:
Fixed